src/ControlzEx/WindowChromeWindow.cs

```diff
@@ -28,6 +26,9 @@ public partial class WindowChromeWindow : Window
         static WindowChromeWindow()
         {
             DefaultStyleKeyProperty.OverrideMetadata(typeof(WindowChromeWindow), new FrameworkPropertyMetadata(typeof(WindowChromeWindow)));
+
+            RenderOptions.ClearTypeHintProperty.OverrideMetadata(typeof(FrameworkElement), new FrameworkPropertyMetadata { DefaultValue = ClearTypeHint.Enabled });
```

Comment:
We shouldn't set those here.
We should better add options to the showcase app for those.